### ISSUE TITLE
return softmax scores instead of logits

### ISSUE THREAD

**Issue by tfriedel** [AI4Bharat/IndicLID#2]
I noticed that if scores were calculated with FTR or FTN
they tend to be between 0 and 1. Well actually I also saw 1.000041.

The scores by IndicLID-BERT however are raw logits, so could for example be 6.42. 
This is unfortunate since as a user you would rather have something resembling a probability and you want scores from the different models to be comparable. I suggest you apply a softmax on the logits in IndicLID-BERT and return those.

**Comment by ammar-wysa** [comment]
Hi @tfriedel, yes, this makes perfect sense! I’m currently working on using this model in an ensemble with others but am having difficulty dealing with the raw logits
